scylla/README.md

```diff
@@ -0,0 +1,249 @@
+
+Consistency levels do not change consistency model or its strongness.
+Even with `-p scylla.writeconsistencylevel=ONE` the data will be written
+according to the number of a table replication factor (RF). Usually,
+by default RF = 3. By using `-p scylla.writeconsistencylevel=ONE` you
+can omit waiting all replicas to write the value. It will improve your
+latency picture a bit but would not affect utilization.
+
+Remember that you can't measure CPU utilization with Scylla by normal
+Unix tools. Check out Scylla own metrics to see real reactors utilization.
+
+Always use [token aware](https://www.scylladb.com/2019/03/27/best-practices-for-scylla-applications/)
+load balancing `-p scylla.tokenaware=true`.
+
+For best performance it is crucial to evenly load all available shards.
+
+### 5. Expected performance target
+
+You can expect about 12500 uOPS / core (shard), where uOPS are basic
+reads and writes operations post replication. Don't forget that usually
+`Core = 2 * vCPU` for HT systems.
+
+For example if we insert a row with RF = 3 we can count at least 3 writes -
+1 write per each replica. That is 1 Transaction = 3 u operations.
+
+Formula for evaluating performance with respect to workloads is:
+
+    uOPS / vCPU = [
+        Transactions * Writes_Ratio * Replication_Factor +
+        Transactions * Read_Ratio   * Read_Consistency_level
+        ] / [ (vCPU_per_node - 2) * (nodes) ]
+
+    where Transactions == `-target` parameter (target throughput).
+
+For example for _workloada_ that is 50/50 reads and writes for a cluster
+of 3 nodes of i3.4xlarge (16 vCPU per node) and target of 120000 is:
+
+    [ 120K * 0.5 * 3 + 120K * 0.5 * 2 (QUORUM) ] / [ (16 - 2) * 3 nodes ] =
+    = 7142 uOPS / vCPU ~ 14000 uOPS / Core.
+
+## Scylla configuration parameters
+
+- `scylla.hosts` (**required**)
+  - A list of Scylla nodes to connect to.
+  - No default. Usage: `-p scylla.hosts=ip1,ip2,ip3,...`
+
+* `scylla.port`
+
+  - CQL port for communicating with Scylla cluster.
+    This port must be the same on all cluster nodes.
+  - Default is `9042`.
+
+- `scylla.keyspace`
+
+  - keyspace name - must match the keyspace for the table created (see above).
+  See https://docs.scylladb.com/getting-started/ddl/#create-keyspace-statement for details.
+  - Default value is `ycsb`
+
+- `scylla.username` and `scylla.password`
+
+  - Optional user name and password for authentication.
+  - See https://docs.scylladb.com/operating-scylla/security/enable-authorization/ for details.
+
+* `scylla.readconsistencylevel`
+* `scylla.writeconsistencylevel`
+
+  * Default value is `QUORUM`
+  - Consistency level for reads and writes, respectively. 
+    See the [Scylla documentation](https://docs.scylladb.com/glossary/#term-consistency-level-any) for details.
+  * *Note that the default setting does not provide durability in the face of node failure. Changing this setting will affect observed performance.* See also `replication_factor`, above.
+
```

Comment:
is this note still accurate? quorum with replication factor >= 3 should provide durability in the face of node failure

Reply:
yeah, I forgot to remove it.

Reply:
removed line